src/index.html

```diff
@@ -3,11 +3,182 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>MyBIKE</title>
+    <link rel="icon" type="image/x-icon"
+    href="./images/logo-dark.svg">
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body page__body--with-menu">
+    <header class="header">
+      <div class="container">
+        <div class="header__content">
+          <div class="header__top">
+            <a href="/" class="header__home-link">
```

Comment:
This link points to nowhere.
![Screenshot 2023-03-06 at 10 41 30](https://user-images.githubusercontent.com/54940595/223059799-bfa32fa1-b39d-416c-929b-10a7b55c57a2.png)